ddclient

```diff
@@ -3687,16 +3687,16 @@ The 'namecheap' protocol is used by DNS service offered by www.namecheap.com.
 Configuration variables applicable to the 'namecheap' protocol are:
   protocol=namecheap           ##
   server=fqdn.of.service       ## defaults to dynamicdns.park-your-domain.com
-  login=service-login          ## login name and password  registered with the service
-  password=service-password    ##
-  fully.qualified.host         ## the hostname to update.
+  login=service-login          ## example.com to update
```

Comment:
```suggestion
  login=service-login          ## login name for your dynamic DNS record
```

I am not sure if this is the best variant but your suggestion is unclearer than the one we have right now.

Reply:
yes